### ISSUE TITLE
Build fails: No value passed for parameter 'p9' in AppViewModel.kt and type mismatches

### ISSUE THREAD

**Issue by alevchuk** [ACINQ/phoenix#67]
Following https://github.com/ACINQ/phoenix/blob/master/BUILD.md

I get Build failed with 3 errors and 37 warnings. The errors are:
1. fr/acinq/phoenix/AppViewModel.kt: (413, 52): No value passed for parameter 'p9'
2. fr/acinq/phoenix/AppViewModel.kt: (484, 32): Type mismatch: inferred type is Future<ChannelCommandResponse!>! but Future<String> was expected
3. fr/acinq/phoenix/AppViewModel.kt: (502, 32): Type mismatch: inferred type is Future<ChannelCommandResponse!>! but Future<String> was expected

Side note:
- eclair build instructions produce snapshots so I had to change `def eclair_version = "0.3.8-android-phoenix"` in build.gradle to `def eclair_version = "0.3.9-android-phoenix-SNAPSHOT"`

**Comment by alevchuk** [comment]
My environment is latest stable versions for Studio, JDK, and Maven: 
```
Android Studio 3.6.3
Build #AI-192.7142.36.36.6392135, built on April 13, 2020
Runtime version: 1.8.0_212-release-1586-b4-5784211 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
macOS 10.15.3
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 12
Non-Bundled Plugins: org.jetbrains.kotlin, com.google.services.firebase

mvn -v
Apache Maven 3.6.3 (cecedd343002696d0abb50b32b541b8a6ba2883f)
Maven home: /Users/user/src/apache-maven-3.6.3
Java version: 11.0.7, vendor: AdoptOpenJDK, runtime: /Library/Java/JavaVirtualMachines/adoptopenjdk-11.jdk/Contents/Home
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.15.3", arch: "x86_64", family: "mac"
```

**Comment by dpad85** [comment]
Hi that's because by updating the version of eclair-core to `0.3.9-android-phoenix-SNAPSHOT`, a new field in `SendPaymentRequest` is added and the return type of `Eclair.close` is modified, so a few fixes are needed in the frontend side.

This is fixed by 2829df72b4cd30a0b46aaf41dd18e30438e4a954
